doc/manual.rst

```diff
@@ -90,6 +92,16 @@ Resampling strategies
 
 Examples for using holdout and cross-validation can be found in `auto-sklearn/examples/ <examples/>`_
 
+Ensemble Building Process
+======================
+
+*auto-sklearn* uses ensemble selection by `Caruana et al. (2004) <https://dl.acm.org/doi/pdf/10.1145/1015330.1015432>`_ 
+to build an ensemble based on the models’ prediction for the validation set. The following hyperparameters control how the ensemble is constructed:
+
+* ``ensemble_size`` determines the maximal size of the ensemble. if it is set to zero, no ensemble will be constructed.
+* ``ensemble_nbest`` allows the user to directly specify the number of models used for the ensemble.  This hyperparameter can be an integer *n*, such that only the best *n* models are used in the final ensemble. If a float between 0.0 and 1.0 is provided, ``ensemble_nbest`` would be interpreted as a fraction suggesting the percentage of models to use in the ensemble building process.
```

Comment:
I wonder if one should write 'number of models considered for the ensemble' instead of 'used for the ensemble' as there is no guarantee that they end up in the final ensemble.

Reply:
Also, could you please add that if ensemble_nbest is a float, this is library pruning as described in https://dl.acm.org/doi/10.1109/ICDM.2006.76